tower-sessions-core/src/session_store.rs

```diff
@@ -64,6 +67,7 @@ pub type Result<T> = std::result::Result<T, Error>;
 ///     }
 /// }
 /// ```
+#[cfg_attr(test, automock)]
```

Comment:
Rather than creating visual noise in the library code, I opted to create mocks directly in the test modulate here:

https://github.com/maxcountryman/tower-sessions/blob/6ad8933b4f5e71f3202f0c1a28f194f3db5234c8/tower-sessions-core/src/session_store.rs#L301-L337

This also has the benefit of allowing for alterations of the mocks (as seen here with the colliding store, for example).

Reply:
My intention was to avoid the duplicate interface definitions. However, I am not gonna tell you how to run your project.

This seems fine.